docs/source/guides/contributor/chapters/plugins.rst

```diff
@@ -77,3 +77,62 @@ definition itself, in this case, ``jobscripts``.
 
 To summarize, still using the same example, the fully qualified Avocado plugin
 name is going to be ``job.prepost.jobscripts``.
+
+
+Plugin config files
+~~~~~~~~~~~~~~~~~~~
+
+There are two ways to extend settings of extra plugin configuration. Plugins
```

Comment:
Suggestions:

1. go straight to the point, that is, drop most of this paragraph;
2. start with `/etc/avocado/conf.d`, which is easier to understand and will solve 99.9% of users' needs

Reply:
This was already there, I just moved places, but sure. I will fix it.